plugins/GSdx/Renderers/DX11/GSDevice11.cpp

```diff
@@ -1468,7 +1467,7 @@ void GSDevice11::CreateShader(std::vector<char> source, const char* fn, ID3DIncl
 	}
 }
 
-void GSDevice11::CreateShader(std::vector<char> source, const char* fn, ID3DInclude *include, const char* entry, D3D_SHADER_MACRO* macro, ID3D11PixelShader** ps)
+void GSDevice11::CreateShader(const std::vector<char> source, const char* fn, ID3DInclude *include, const char* entry, D3D_SHADER_MACRO* macro, ID3D11PixelShader** ps)
```

Comment:
Would `const std::vector<char>&` be better?

Reply:
Probably, since it'd use up space equivalent to a pointer rather than an entire new vector object.
Codacy would most likely state to make it so anyways.